src/main/scala/nl/knaw/dans/easy/multideposit/CommandLineOptions.scala

```diff
@@ -43,7 +43,7 @@ object CommandLineOptions extends DebugEnhancedLogging {
       outputDepositDir = opts.outputDepositDir().getAbsoluteFile,
       datamanager = opts.datamanager(),
       depositPermissions = DepositPermissions(props.getString("deposit.permissions.access"), props.getString("deposit.permissions.group")),
-      formatsFile = new File(props.getString("formats.file")),
+      formatsFile = new File(homeDir, "cfg/formats.txt"),
```

Comment:
I was just hesitating about this, but you're right. This is a config file, so indeed the application can assume that it is in the config directory. This is btw also consistent with line 34, where we read the properties in a similar fashion. Do you think it is wise to add a comment above line 46 saying something like "_formats.txt is a configuration file. The application may therefore assume that it is the config directory._", or do you think that's not necessary?